packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -2,11 +2,12 @@ import { BaseAssetId } from '@fuel-ts/address/configs';
 import { toHex } from '@fuel-ts/math';
 import { Provider } from '@fuel-ts/providers';
 import { Signer } from '@fuel-ts/signer';
+import { defaultChainConfig, defaultConsensusKey } from '@fuel-ts/utils';
 import type { ChildProcessWithoutNullStreams } from 'child_process';
 import { spawn } from 'child_process';
 import { randomUUID } from 'crypto';
 import { hexlify } from 'ethers';
-import fsSync from 'fs';
+import fsSync, { existsSync } from 'fs';
```

Comment:
We can DRY this import:

> _You'll need to replace `fsSync.existsSync` and `fsSync.mkdirSync` usages below._

```suggestion
import { existsSync, mkdirSync } from 'fs';
```